difflint/data/eslint_terse_reporter.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Format errors for consumption by a lint-diff tool that compares JSHint output
+ * between two revisions of the same file. This reporter is not intended for
+ * human consumption.
+ */
+
+var formatMessage = function (filename, message) {
+    var messageType = "warning";
+
+    // These criteria match the criteria for displaying "error" in the default
+    // reporter for eslint.
+    if (message.fatal || message.severity === 2) {
+        messageType = "error";
+    }
+    
+    return filename + "|" + messageType + "|" + message.message;
+};
+
+var compareMessages = function (filename, messageA, messageB) {
+    var formattedMessageA = formatMessage(filename, messageA);
+    var formattedMessageB = formatMessage(filename, messageB);
+    return formattedMessageA.localeCompare(formattedMessageB);
```

Comment:
Is locale compare really what you want here? The output of this tool is read by another tool so I think it would be better to have consistent output across locales.